test/framework/src/main/java/org/elasticsearch/test/InternalTestCluster.java

```diff
@@ -608,6 +610,11 @@ private static Settings getRandomNodeSettings(long seed) {
             builder.put(INITIAL_STATE_TIMEOUT_SETTING.getKey(), "0s");
         }
 
+        if (usually()) {
+            // Disable fsync in most test runs to speed up tests
+            builder.put(IndexModule.NODE_STORE_USE_FSYNC.getKey(), false);
```

Comment:
This is a neat speedup for internal cluster tests on my Mac. I don't see why we wouldn't usually turn off fsync to give people faster builds and less SSD wear even though this is the uncommon path in prod (maybe :)).